method/SEDR/config/config_stereoseq.json

```diff
@@ -0,0 +1,7 @@
+{
+    "n":6,
+    "cluster_method": "mclust", 
+    "using_dec": true, 
+    "HVG": false,
+    "reference": "https://sedr.readthedocs.io/en/latest/Tutorial4_Stereo-seq.html"
```

Comment:
I believe we call it "source" elsewhere, but I guess the word used is not greatly important.